modules/nf-core/tximeta/tximport/meta.yml

```diff
@@ -25,9 +25,7 @@ input:
         description: |
           Groovy Map containing information related to the experiment as a whole
           e.g. `[ id:'SRP123456' ]`
-    - '"quants/*"':
-        type: directory
-        description: Directory containing quantification files
+    - quants/*: {}
```

Comment:
What happened here?

Reply:
Will merge this, but fix in the modules repo for a future update